components/slidingPanel/slidingPanel.js

```diff
@@ -283,5 +284,11 @@ export default class SlidingPanel extends Component {
         </div>
       </div>
     );
+
+    return (
```

Comment:
this doesn't seem optional nor backward compatible as it is described

Reply:
Ricardo proposed using Global always with SlidingPanel